### DISCUSSION TITLE
What is Outstanding Tasks

### DISCUSSION THREAD

**Discussion by linonetwo** [bpmnServer/bpmn-server#168] [Q&A]
Seems not a standard bpmn vocab, I see this in http://localhost:3000/bpmn-server/customization#use-case-2-invoking-an-outstanding-tasks

**Comment by linonetwo** [comment]
Seems means "restored previous tasks"?

**Comment by ralphhanna** [comment]
Sorry, I don’t understand the question 
Can you please elaborate

**Reply by linonetwo** [reply]
Sure, just see my recent PR. The doc I am changing has word "outstanding", I don't know what it means in bpmn context.

**Comment by linonetwo** [comment]
>  Outstanding in this case means User Tasks with status of wait

https://github.com/bpmnServer/bpmn-server/pull/169
